src/Components/Blazor/Directory.Build.props

```diff
@@ -2,9 +2,23 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
 
   <PropertyGroup>
-    <ComponentsPackageVersion>$(PackageVersion)</ComponentsPackageVersion>
+    <!-- Out-of-band release -->
+    <UseProjectReferences>false</UseProjectReferences>
+    <ComponentsPackageVersion>3.0.0-preview3-19153-02</ComponentsPackageVersion>
```

Comment:
**TODO:** Determine the final `ComponentsPackageVersion`. It needs to be whatever ships as Preview 3.